src/main/java/com/softteam/vocabuilder/service/CategoryServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.softteam.vocabuilder.service;
+
+import com.softteam.vocabuilder.persistence.entity.Category;
+import com.softteam.vocabuilder.persistence.repository.CategoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class CategoryServiceImpl implements ICategoryService{
+    @Autowired
+    private CategoryRepository categoryRepository;
+    @Override
+    public Category create(Category category) {
+        return categoryRepository.save(category);
+    }
+
+    @Override
+    public void update(Category category) {
+        System.out.println("Service");
```

Comment:
should this line be here or we can remove it?

Reply:
was a proof and i forgot delete it.